### ISSUE TITLE
Improved ".nlogo" file format

### ISSUE THREAD

**Issue by mrerrormessage** [NetLogo/NetLogo#849]
Ongoing discussion about whether this should be in yaml or xml.

**Comment by qiemem** [comment]
I believe we left off an in-person meeting with "let's create actual examples of different possibilities and compare" rather than just speculatively arguing the abstract merits of different formats.

**Comment by mrerrormessage** [comment]
That's correct. I was going to update this ticket and ask both you and @frankduncan to post representative examples in yaml and xml, respectively.

Criteria:

* Must contain all fields currently in a NetLogo file
* Must be editable in a text editor

Other than that, I think it's a question of format and relative merit. Whatever it is that needs to be done, we'll get it done.

**Comment by Idloj** [comment]
That's a cool quote for life, @mrerrormessage: "Whatever it is that needs to be done, we'll get it done."

**Comment by mrerrormessage** [comment]
[Translated a representative section of WSP into xml this morning](https://gist.github.com/mrerrormessage/b5476c3e4c887fb45f4c ), I would appreciate feedback if @qiemem or @frankduncan have time. Leave any comments on the gist, please, and I will update the file if there are any suggestions. Also, if anyone would care to write up an alternate format I can add it to the gist (or the gist can be forked) for a side-by-side comparison.

**Comment by alan-isaac** [comment]
Is it still the intention for these changes to include allowing users to hide/remove the View via the Settings dialogue, as suggested by Seth long ago?

**Comment by SethTisue** [comment]
@alan-isaac that doesn't seem on-topic for this ticket?

**Comment by SethTisue** [comment]
ah, I see. yes, #849 going forward would unblock #681, but no, that doesn't mean that #849 going forward would automatically mean #681 would happen as well. these are separate, only tangentially related issues.

**Comment by SethTisue** [comment]
some notes on a possible new format: https://github.com/NetLogo/NetLogo/wiki/The-nlogox-format

**Comment by ERSUCC** [comment]
Models are now saved as XML as of NetLogo 7. Closing the issue as completed.
